common/sai_npu.py

```diff
@@ -370,6 +370,8 @@ def hostif_dataplane_start(self, ifaces):
         for inum, iname in ifaces.items():
             socket_addr = 'tcp://{}:10001'.format(self.server_ip)
             self.hostif_map[(0, int(inum))] = socket_addr
+            # If iface is not up, init will hang
```

Comment:
It's known vslib issue
https://github.com/opencomputeproject/SAI-Challenger/blob/573b2ba6f8d47218af575d8de580252a74a5051b/tests/ut/test_hostif.py#L15
prefer to keep this code gereric
this may help to catch real issue on HW testbed